packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Vertical.Grow.Example.tsx

```diff
@@ -21,10 +21,11 @@ export class VerticalStackGrowExample extends React.Component<{}, {}> {
     });
 
     const stackTokens: IStackTokens = { childrenGap: 5 };
+    const tenPaddingTokens: IStackTokens = { padding: 10 };
 
     return (
       <Stack tokens={stackTokens}>
-        <Stack tokens={stackTokens} padding={10} className={styles.root}>
+        <Stack className={styles.root} tokens={{ ...stackTokens, ...tenPaddingTokens }}>
```

Comment:
This will create a new object each time. Can `tenPaddingTokens` also contain `stackTokens` when initialized? Minor, but I think we should encourage good practices as reasonably as possible in the examples.